sdk/src/main/java/com/hedera/hashgraph/sdk/FeeData.java

```diff
@@ -0,0 +1,99 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import javax.annotation.Nullable;
+
+public class FeeData {
+
+    @Nullable
+    private FeeComponents nodeData;
+    @Nullable
+    private FeeComponents networkData;
+    @Nullable
+    private FeeComponents serviceData;
+
+    public FeeData() {
+        nodeData = null;
+        networkData = null;
+        serviceData = null;
+    }
+
+
+
+
+
+    // getters
+
+    boolean hasNodeData() {
+        return nodeData != null;
+    }
+    boolean hasNetworkData() {
+        return networkData != null;
+    }
+    boolean hasServiceData() {
+        return serviceData != null;
+    }
+    FeeComponents getNodeData() {
```

Comment:
As aforementioned use groups of get/set methods per field instead of all getters followed by all setters. Also, add one new line between each method.